### PR TITLE
support setting affinity when using JindoFSX Dataload

### LINKED ISSUE #2758: [FEATURES] JindoFSx dataload support affinity
**What feature you'd like to add:**
JindoFSx dataload support affinity

### PR DESCRIPTION
**Pull request by allenhaozi**
Signed-off-by: mahao <allenhaozi@gmail.com>

### Ⅰ. Describe what this PR does
support setting affinity when using JindoFSX Dataload

### Ⅱ. Does this pull request fix one issue?

fixes #2758

### CODE REVIEW

File: charts/fluid-dataloader/jindofsx/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+### 0.9.0
+- Support for setting affinity,tolerations,nodeSelector,schedulerName
```
Comment (TrafalgarZZZ) [inline charts/fluid-dataloader/jindofsx/CHANGELOG.md:2]: Suggest to use `0.6.0` or `0.5.1` to indicate the Chart changes

Comment (allenhaozi) [reply charts/fluid-dataloader/jindofsx/CHANGELOG.md]: done

**Review by TrafalgarZZZ — COMMENTED** [review]
/lgtm

**Review by cheyang — APPROVED** [review]

**Pull request merged**